scripts/emulator-tests/fixtures.ts

```diff
@@ -137,7 +137,7 @@ rules_version = '2';
 service firebase.storage {
   match /b/{bucket}/o {
     match /{allPaths=**} {
-      allow read, write: if math.abs(-10) == 10;
+      allow read, write: if math.abs(-10) === 10;
```

Comment:
I'm not sure if `===` is valid in security rules, does this pass integration tests?

Reply:
Done.